core/src/apps/bitcoin/sign_tx/layout.py

```
@@ -262,3 +256,26 @@ async def confirm_nondefault_locktime(
         br_code=ButtonRequestType.SignTx,
         description=description,
     )
+
+
+async def confirm_op_return(
+    ctx: wire.Context, data: bytes, amount: int, coin: CoinInfo
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)